src/posix/client.cpp

```diff
@@ -111,7 +120,7 @@ static bool DoWrite(int aFile, const void *aBuffer, size_t aSize)
     return ret;
 }
 
-static int ConnectSession(void)
```

Comment:
why do we remove `static`?

Reply:
We use `static` to avoid global function conflicts. Now they are all inside of `ot::Client` namespace.